screens/Home/TopPicksCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC } from "react";
+import { Image, ScrollView, Text, TouchableOpacity, View } from "react-native";
+import movieCards from "../../assets/MovieCardImages";
+import CommonText from "../../shared/CommonText";
```

Comment:
can we use absolute imports instead: 

for example rather than have have `../../shared/CommonText` we would have something like `@src/shared/CommonText` 

For reference see the following articles:

- https://mobiledevtutorials.com/absolute-imports-in-react-native/
- https://javascript.plainenglish.io/react-native-absolute-imports-quick-easy-5e3b60897f34


You can browse for more options